packages/mc-scripts/package.json

```diff
@@ -42,7 +42,7 @@
     "@commercetools-frontend/mc-html-template": "16.3.0",
     "@svgr/webpack": "5.2.0",
     "babel-loader": "8.0.6",
-    "babel-plugin-react-intl": "5.1.18",
+    "babel-plugin-react-intl": "6.0.1",
```

Comment:
<img width="674" alt="CleanShot 2020-03-09 at 08 45 09@2x" src="https://user-images.githubusercontent.com/1877073/76192524-4e287980-61e2-11ea-9c99-1df259da543f.png">

So we should check the verion of that I suppose?

Reply:
Have you tried running the `extract-intl` command?

Reply:
Yes, it worked for me.

```txt
» y i18n:build

$ ./packages/mc-scripts/bin/mc-scripts.js extract-intl --output-path=$(pwd)/packages/i18n/data 'packages/**/!(*.spec|*.d).{js,ts,tsx}'

Storing language files in memory
Run extraction on all files
Writing core translation messages to: /Users/tdeekens/Development/merchant-center-application-kit/packages/i18n/data/core.json
```